### PR TITLE
Support for CsvConverters to directly provide JSON schema

### PR DESCRIPTION
**Pull request by KhaninArtur**
To implement new templates that will use CSV as an input source, we suggest extending the functionality of the `CsvConverters` class.

This PR makes it possible to provide JSON schema of the input CSV lines via String.

### CODE REVIEW

File: v2/common/src/main/java/com/google/cloud/teleport/v2/transforms/CsvConverters.java [hunk 1]
```diff
@@ -293,9 +296,14 @@ public PCollectionTuple expand(PCollectionTuple lines) {
       }
 
       // If no udf then use json schema
-      if (jsonSchemaPath() != null) {
+      if (jsonSchemaPath() != null || jsonSchema() != null) {
 
-        String schema = SchemaUtils.getGcsFileAsString(jsonSchemaPath());
+        String schema;
```
Comment (prathapreddy123) [inline v2/common/src/main/java/com/google/cloud/teleport/v2/transforms/CsvConverters.java:301]:
```suggestion
 String schema = jsonSchemaPath() != null ? SchemaUtils.getGcsFileAsString(jsonSchemaPath()) : jsonSchema();
```

**Review by prathapreddy123 — COMMENTED** [review]
Which module is using  jsonSchema instead of jsonSchemaPath?

**Comment by KhaninArtur** [comment]
> Which module is using jsonSchema instead of jsonSchemaPath?

@prathapreddy123 the template from [this PR](https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/210) will use it.

**Comment by prathapreddy123** [comment]
> > Which module is using jsonSchema instead of jsonSchemaPath?
> 
> @prathapreddy123 the template from [this PR](https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/210) will use it.

Great.  LGTM.   Please rebase the branch to latest master.

**Comment by KhaninArtur** [comment]
@prathapreddy123 thanks! Rebased.

**Pull request merged**